app/controllers/api/v2/people_controller.rb

```diff
@@ -3,16 +3,23 @@
 module Api
   module V2
     class PeopleController < ApiController
+      def index
+        people = ::V2::People::Finder.new(filter_params).call
+
+        paginate people, include: included_relationships, each_serializer: ::V2::PersonSerializer
+      end
+
       def create
         person = Person.create(person_attributes)
-
         render json: person, status: :created, include: included_relationships, serializer: ::V2::PersonSerializer
       end
 
-      def index
-        people = ::V2::People::Finder.new(filter_params).call
+      def update
+        person = Person.find(params[:id])
 
-        paginate people, include: included_relationships, each_serializer: ::V2::PersonSerializer
+        person.update(person_attributes)
```

Comment:
Can we use `update!` instead of `update` so an exception is raised if the record is invalid

Reply:
sure, but consider that there is not any validation now, I have verified with Jonny about this.

Reply:
I think this is slightly different, let's catchup in person